action.yml

```diff
@@ -43,6 +43,12 @@ inputs:
   skip_api_build:
     description: "Skips the build step for the Azure Functions source code if set to true."
     required: false
+  production_branch:
+    description: "Production branch. Deployments from other branches will be considered preview environments."
+    required: false
+  deployment_environment:
+    description: "Explicit name of Azure Static Web Apps environment for this deployment."
```

Comment:
I feel like this should be something like "Name of the Azure Static Web Apps environment to deploy to"